openquakeplatform/settings.py

```diff
@@ -5,4 +5,4 @@
 import geonode
 
 from geonode.settings import *
-from openquakeplatform.local_settings import *
+#from openquakeplatform.local_settings import *
```

Comment:
If not jet needed remove it.

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/3a0f5d6ec0cb18202936596ab6b999e188b18c72